app/src/js/components/Collections/overview.js

```diff
@@ -113,220 +108,184 @@ class CollectionOverview extends React.Component {
-                    search: getPersistentQueryParams(location),
-                    state: {
-                      name: collectionName,
-                      version: collectionVersion,
-                    },
-                  })}
-                >
-                  Copy
-                </Link>
-              </li>
-              <li>
-                <Link
-                  className="button button--edit button--small button--green"
-                  to={(location) => ({
-                    pathname: `/collections/edit/${collectionName}/${collectionVersion}`,
-                    search: getPersistentQueryParams(location),
-                  })}
-                >
-                  Edit
-                </Link>
-              </li>
-              <li>{this.renderDeleteButton()}</li>
-            </ul>
-            <span className="last-update">
-              {lastUpdated(get(record, 'data.timestamp'))}
+  return (
+    <div className="page__component">
+      <Helmet>
+        <title> Collection Overview </title>
+      </Helmet>
+      <section className="page__section page__section__controls">
+        <div className="collection__options--top">
+          <ul>
+            <li>
+              <Breadcrumbs config={breadcrumbConfig} />
+            </li>
+            <li>
+              <div className="dropdown__collection form-group__element--right">
+                <SimpleDropdown
+                  className='collection-chooser'
+                  label={'Collection'}
+                  title={'Collections Dropdown'}
+                  value={collectionId}
+                  options={sortedCollectionIds}
+                  id={'collection-chooser'}
+                  onChange={changeCollection}
+                />
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section className="page__section page__section__header-wrapper">
+        <div className="heading-group">
+          <ul className="heading-form-group--left">
+            <li>
+              <h1 className="heading--large heading--shared-content with-description">
+                {strings.collection}: {collectionLabelForId(collectionId)}
+              </h1>
+            </li>
+            <li>
+              <Link
+                className="button button--copy button--small button--green"
+                to={(location) => ({
+                  pathname: '/collections/add',
+                  search: getPersistentQueryParams(location),
+                  state: {
+                    name: collectionName,
+                    version: decodedVersion,
+                  },
+                })}
+              >
+                Copy
+              </Link>
+            </li>
+            <li>
+              <Link
+                className="button button--edit button--small button--green"
+                to={(location) => ({
+                  pathname: `/collections/edit/${collectionName}/${collectionVersion}`,
```

Comment:
This pathname is not encoded, why?

Reply:
In this case, `collectionVersion` is taken from the params, so it is already encoded